Document `sum` statistics for sensors with state_class measurement (#1045)
home-assistant/developers.home-assistant: docs/core/entity/sensor.md

| Type | Description
| ---- | -----------
| measurement | The state represents _a measurement in present time_, not a historical aggregation such as statistics or a prediction of the future. Examples of what should be classified `measurement` are: current temperature, accumulated energy consumption, accumulated cost.  Examples of what should not be classified as `measurement`: Forecasted temperature for tomorrow, yesterday's energy consumption or anything else that doesn't include the _current_ measurement. For supported sensors, statistics of hourly min, max and average sensor readings or of the accumulated growth or decline of the sensor's value since it was first added is updated hourly.
| total_increasing | Similar to `measurement`, with the restriction that the state represents a monotonically increasing total, e.g. an amount of consumed gas, water or energy. Statistics of the accumulated growth of the sensor's value since it was first added is updated hourly.

## Long-term Statistics

Home Assistant has support for storing sensors as long-term statistics if the entity has

> the right properties. To opt-in for statistics is that the sensor has

Author: ```suggestion
the right properties. To opt-in for statistics, the sensor must have
```

`state_class` set to one of the valid state classes: `measurement` or
`total_increasing`.

> ### Value entities - entities not representing a total amount

Reviewer: `Value entities` is ambiguous, I think. Value is a generic term in computer science talk.

Home Assistant tracks the min, max and mean value during the statistics period. The
`state_class` property must be set to `measurement`, and the `device_class` must not be
either of `energy`, `gas`, or `monetary`

> ### Entities not representing a total amount

Reviewer: This header seems out of place or incorrect? We're talking about total amount below.

|   2021-08-01T15:00:00  |     5  |  15  |
|   2021-08-01T16:00:00  |    10  |  20  |

#### State class `measurement`

> For sensors with state_class `measurement` a total acmount is calculated if:

Reviewer: ```suggestion
For sensors with state_class `measurement` a total amount is calculated if:
```
